README.md

```diff
@@ -39,6 +39,26 @@ Download the latest compiled [release](https://github.com/Nitrokey/opcard-rs/rel
 Plug your Nitrokey 3A Mini and use [nitropy](https://docs.nitrokey.com/software/nitropy/) to install it with 
 `nitropy nk3 update <path/to/release/zip/file>`
 
+## Build Dependencies
+
+`opcard` has these build dependencies:
+- clang
+- libpcsclite
+- nettle
+- pkg-config
+
+To run the tests, you also need these tools:
+- gnupg
+- scdaemon
+- vsmartcard (vpcd)
+
+See the [CI Dockerfile](./ci/Dockerfile) for all steps to set up the build environment on Debian-based distributions.
```

Comment:
We should probably have the actual build instructions here, indicating how to run the tests (with a warning not to do it with another smarcard plugged in, as the tests risk running against it) and maybe pointing to the nitrokey-3-firmware repo for full firmware builds.

Reply:
I pushed a linked to the nitrokey-3-firmware repo here and changed "build environment" to "development environment" as this repo is not really meant to be used as-is and needs to be integrated in a firmware.